skore/src/skore/sklearn/types.py

```diff
@@ -23,6 +23,17 @@
 Aggregate = Union[Literal["mean", "std"], Sequence[Literal["mean", "std"]]]
 
 
+class YPlotData(TypedDict):
```

Comment:
I'm wondering if we should not use a `dataclass` then. Whenever calling the instance, we can access with `.attribute` instead of `["attribute"]`. And they are data :)